draft-ietf-scone-protocol.md

```diff
@@ -317,18 +317,19 @@ A Rate Signal is a 7-bit unsigned integer (0-127). The high six bits are the
 Rate Signal High Bits, and the least significant bit is the most significant
 bit of the Version field.
 
-When sent by a QUIC endpoint, the Rate Signal is set to 127.  Receiving a value
-of 127 indicates that throughput advice is unknown, either because network
-elements on the path are not providing advice or they do not support SCONE. All
-other values (0 through 126) represent the ceiling of rates advised by the
-network element(s) on the path.
+When sent by a QUIC endpoint, the Rate Signal MUST be set to 127.  Receiving a
+value of 127 indicates that throughput advice is unknown or unchanged, either
+because network elements on the path are not providing advice or they do not
+support SCONE.  126 explicitly indicates there is no advised limit. All other
+values (0 through 125) represent the ceiling of rates advised by the network
+element(s) on the path.
```

Comment:
```suggestion
When sent by a QUIC endpoint, the Rate Signal MUST be set to 127.  Receiving a
value of 127 indicates that throughput advice is unknown or unchanged, either
because network elements on the path are not providing new or updated advice or
they do not support SCONE.  126 explicitly indicates there is no advised
limit. All other values (0 through 125) represent the ceiling of rates advised
by the network element(s) on the path.
```